nomad/csi_endpoint.go

```diff
@@ -95,13 +95,16 @@ func (v *CSIVolume) List(args *structs.CSIVolumeListRequest, reply *structs.CSIV
 		return err
 	}
 
-	allowCSIAccess := acl.NamespaceValidator(acl.NamespaceCapabilityCSIAccess)
+	allowVolume := acl.NamespaceValidator(acl.NamespaceCapabilityCSIListVolume,
+		acl.NamespaceCapabilityCSIReadVolume,
+		acl.NamespaceCapabilityCSIMountVolume,
```

Comment:
We probably don't need mount-volume for `List`, do we?

Reply:
`NamespaceValidator` is or-y (that can't be the right way to say that). We wanted mount to imply read which implies list, so I think we want the set of all permissions that should allow access.

Reply:
Gotcha, makes sense.